### ISSUE TITLE
In markdown "chuck item <target>" and "take item <target>" select different things

### ISSUE THREAD

**Issue by maxbruening** [cursorless-dev/cursorless#2384]
In markdown "chuck item <target>" and "take item <target>" select different things at times. I noticed this when I had a file with a markdown list. "chuck item" would drop the first level and second level of the list item. "Take item" would only select the first level of the list. My file looked something like the following:

- This is the first level of a list.
  - This is the second level of a list. 

This may be related to #585

**Comment by pokey** [comment]
I couldn't reproduce this. I had the following markdown file:

```md
- This is the first level of a list.
  - This is the second level of a list. 
```

I verified that "chuck item" and "take item" differed only in the fact that "chuck item" removes of the leading `- ` and "take item" excludes it, which is expected. I tested both the nested and top-level item. Any idea what's different for me here?

**Comment by maxbruening** [comment]
I confirm that the behavior is not present in the file specified as before. However it is present when we add another list item, e.g.:
```
- This is the first level of a list.
  - This is the second level of a list. 
- This is the first level of a list again.
```

I believe it is confusing for users that the presence of a following list item changes the behavior.

**Comment by pokey** [comment]
Yes `"chuck"` and `"take"` should absolutely be consistent, and presence of subsequent list item should not alter behaviour of either. I'll see if I can reproduce
